proposals/0000-publish-arbitrary-data.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: Publish arbitrary data instruction
+Number: 0
+Status: Draft
+Version: 4
+Authors:
+ - Francisco Aguirre
+Created: 2023-11-16
+Impact: Low
+Requires:
+Replaces:
+---
+
+## Summary
+
+This RFC proposes a new instruction, `Publish`, to publish some data to another location.
+
+## Motivation
+
+XCM itself is not a good means of storing and retrieving arbitrary data.
+This fixes the storage part, leaving the retrieval to another mechanism.
+
+## Specification
+
+The new instruction looks like this:
+
+```rust
+Publish { data: BoundedVec<Key, Value, Bound> }
```

Comment:
Could different chains have different values set for `Bound`? And if so, the sending chain should be aware of the `Bound` of the receiving chain, right?